fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_create.vail

```diff
@@ -1,29 +1,8 @@
 package com.vantiq.fhir
-PROCEDURE fhirService.create(type String REQUIRED, resource Object REQUIRED, versionId String, generalParms Object): Object
+PROCEDURE fhirService.create(type String REQUIRED, resource Object REQUIRED, versionId String): Object
```

Comment:
`create` doesn't need general stuff.  The only one supported is `_format` which we don't support anyway.